### ISSUE TITLE
Extracting FlexiPage metadata is missing LWC components.

### ISSUE THREAD

**Issue by NithunGHub** [forcedotcom/cli#1010]
### Summary

Lightning Record Page with LWC components is missing those component instances on metadata extract.

### Steps To Reproduce:

1. create a record Page and place custom LWC.
2. Extract the Flexi Page and the component instance of LWC is not retrieved.

### Expected result

record page should have the component instance of the LWC when extracted.

### Actual result

record page is missing component instance of the LWC when extracted.

**Comment by fpbrault** [comment]
I've also observed this issue, and it does not appear to be specific to sfdx, as it also happens when generating a changeset. I think this is actually a Salesforce issue...

**Comment by shetzel** [comment]
@NithunGHub - can you please provide more details in the repro and what exactly is missing in the file?  Maybe an example file of what you're expecting or the missing field?

**Comment by NithunGHub** [comment]
Any lwc components on flexi record page is missing on extract.

**Comment by shetzel** [comment]
Tracked with W-9281878, with the LWC team.

**Comment by tsalb** [comment]
+1 to being affected. 

For an additional data point, included is a sample flexipage XML of an appExchange component being stripped away. I've confirmed it is not *actually* removed when looking at App Builder in the Summer 21 full sandbox.

![image](https://user-images.githubusercontent.com/32919768/118535405-4a09bd00-b6ff-11eb-8b2b-8f801e2a879c.png)

Output of `sfdx plugins --core` if needed, but sounds like it's not a cli specific issue.
```
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.2 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-update 1.4.0-3 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.9 (core)
auth 1.5.3 (core)
config 1.2.8 (core)
generator 1.1.5 (core)
salesforcedx 51.12.0 (core)
├─ schema 1.0.6 (core)
├─ apex 0.2.0 (core)
├─ data 0.4.6 (core)
├─ limits 1.2.0 (core)
├─ custom-metadata 1.0.12 (core)
├─ user 1.2.10 (core)
├─ templates 51.3.1 (core)
├─ salesforce-alm 51.6.23 (core)
├─ org 1.6.5 (core)
└─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
sfdmu 4.2.6
sfdx-cli 7.101.0 (core)
telemetry 1.2.0 (core)
```

**Comment by dcheng666666** [comment]
I also got this issue. I put one custom lwc component under Tab in a Flexipage, When i try to retrieve the source code for Flexipage by the following command, I don't find any lwc component description in Flexipage xml.

`sfdx force:source:retrieve -x manifest/package.xml`

**Comment by ghost** [comment]
Is it fixed?

On trailblazer it seems to be but not for me:
https://trailblazer.salesforce.com/issues_view?id=a1p4V000001a1N4QAI&title=lwc-component-instances-are-dropped-from-flexipage-in-metadata-api-retrieve

```
$ sfdx plugins --core
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.3.1 (core)
@oclif/plugin-not-found 1.2.6 (core)
@oclif/plugin-plugins 1.10.11 (core)
@oclif/plugin-update 1.5.0 (core)
@oclif/plugin-warn-if-update-available 1.7.3 (core)
@oclif/plugin-which 1.0.4 (core)
@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
alias 1.2.1 (core)
apex 0.8.0 (core)
auth 1.8.1 (core)
community 1.1.4 (core)
config 1.3.19 (core)
custom-metadata 1.0.12 (core)
data 0.6.9 (core)
generator 1.2.2 (core)
info 1.2.1 (core)
limits 1.3.0 (core)
org 1.11.1 (core)
salesforce-alm 53.10.2 (core)
schema 1.1.0 (core)
sfdx-cli 7.138.1 (core)
source 1.8.11 (core)
telemetry 1.4.0 (core)
templates 53.5.0 (core)
trust 1.1.0 (core)
user 1.7.1 (core)
```

**Comment by mdonnalley** [comment]
LWC team has fixed this issue. Please re-open if you think it's still an issue
